scripts/ci-check_release_commit_messages.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+fail=0
+
+git fetch origin develop
+
+# list all commits between HEAD and develop
+commit=$(git rev-list origin/develop..)
+
+commit_id=$(git log --pretty=format:'%H' -n 1 $commit)
+echo "Checking $commit"
+
+# 1. Checking if the commit is in develop"
+
+if [[ $(git merge-base --is-ancestor $commit_id HEAD | grep --only-matching "remotes/origin/develop") == "remotes/origin/develop" ]]; then
+  continue
```

Comment:
You are not in a loop, `continue` does not make sense here nope?

```
if ..; then
  echo ..
  exit 0
exit 1
```

And you can drop the `fail` variable altogether.